### PR TITLE
[LOGO][macOS] Updates "retro" style to a cleaner one

### PR DESCRIPTION
**Pull request by GentleHoneyLover**
I think this logo (from original archey) is much cleaner and slicker ;)

> see #98.

### CODE REVIEW

**Comment by GentleHoneyLover** [comment]
This is how it'll look like:

Screen Shot 2021-04-27 at 21 18 11

**Comment by HorlogeSkynet** [comment]
Thanks for the PR ! I don't mind you editing the new "retro" one (not yet published), but I would rather you don't change the "default" one, without proper reviews from other users. From relatives, it corresponds well the way it is. Bye 👋

**Comment by GentleHoneyLover** [comment]
@HorlogeSkynet, you're the boss :)) Updating just the retro one is fine for me...

Should I open a new PR for that?

**Comment by HorlogeSkynet** [comment]
Great 👌 you may only add a commit to your own branch, it would update this PR automatically !

File: archey/logos/darwin.py [hunk 1]
```diff
@@ -38,21 +38,17 @@
...
-    """{c[0]}  {c[4]}kMMMMMMMMMMMMMMMMMMMMMMMMWd.{c[0]}    """,
-    """{c[0]}  {c[5]}.XMMMMMMMMMMMMMMMMMMMMMMMMMMk{c[0]}   """,
-    """{c[0]}   {c[5]}.XMMMMMMMMMMMMMMMMMMMMMMMMK.{c[0]}   """,
-    """{c[0]}     {c[6]}kMMMMMMMMMMMMMMMMMMMMMMd{c[0]}     """,
-    """{c[0]}      {c[6]};KMMMMMMMWXXWMMMMMMMk.{c[0]}      """,
-    """{c[0]}        {c[6]}.cooc,.{c[0]}    {c[6]}.,coo:.{c[0]}        """
+    """{c[0]}                {c[1]}###{c[0]}               """,
+    """{c[0]}              {c[1]}####{c[0]}                """,
+    """{c[0]}              {c[1]}###{c[0]}                 """,
+    """{c[0]}      {c[1]}#######    #######{c[0]}          """,
```
Comment (HorlogeSkynet) [inline archey/logos/darwin.py:44]: For "inverted-selection" consistency :

```suggestion
    """{c[0]}      {c[1]}#######{c[0]}    {c[1]}#######{c[0]}          """,
```

File: archey/logos/darwin.py [hunk 2]
```diff
@@ -38,21 +38,17 @@
...
+    """{c[0]}    {c[2]}######################{c[0]}        """,
+    """{c[0]}   {c[2]}#####################{c[0]}          """,
+    """{c[0]}   {c[4]}####################{c[0]}           """,
+    """{c[0]}   {c[4]}####################{c[0]}           """,
+    """{c[0]}   {c[4]}#####################{c[0]}          """,
+    """{c[0]}    {c[5]}######################{c[0]}        """,
+    """{c[0]}     {c[5]}####################{c[0]}         """,
+    """{c[0]}       {c[6]}################{c[0]}           """,
+    """{c[0]}        {c[6]}####     #####{c[0]}            """,
```
Comment (HorlogeSkynet) [inline archey/logos/darwin.py:53]: Same :

```suggestion
    """{c[0]}        {c[6]}####{c[0]}     {c[6]}#####{c[0]}            """,
```

**Review by HorlogeSkynet — CHANGES_REQUESTED** [review]

**Review by HorlogeSkynet — APPROVED** [review]

**Comment by parkerbxyz** [comment]
👋 Can this option (`--logo-style`) be set in a configuration file so it doesn't have to be included as an argument each time?

**Comment by HorlogeSkynet** [comment]
Hi @parkerbxyz, yes it could. I'll add this to the TODO-list.

**Comment by ingrinder** [comment]
See #124 - I would attach it to the card in projects but I'm not entirely sure how...!

**Pull request merged**